apps/demo/src/components/pages/DetailsListPage.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import { DemoPage } from '../DemoPage';
+import { DetailsListPageProps } from 'office-ui-fabric-react/lib/components/DetailsList/DetailsList.doc';
+
+export const DetailsListPage = (props: { isHeaderVisible: boolean }) => (
+  <DemoPage {...{ ...DetailsListPageProps, ...props }} />
```

Comment:
Separating this code out is revealing a repetitive pattern in most of these files.. can this be reduced?

Reply:
If there's a way to do that, I'd reserve that in Part 4 - since any changes to do that will likely require lots of refactoring in fabric-websites beyond what I've done.